src/handlers/beta_backport.rs

```diff
@@ -0,0 +1,52 @@
+use crate::github::{Event, IssuesAction, Label};
+use crate::handlers::Context;
+use regex::Regex;
+
+lazy_static! {
+    // See https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue
+    // Max 19 digits long to prevent u64 overflow
+    static ref CLOSES_ISSUE: Regex = Regex::new("(close[sd]|fix(e[sd])?|resolve[sd]) #(\\d{1,19})").unwrap();
+}
+
+pub(crate) async fn handle(
+    ctx: &Context,
+    event: &Event,
+) -> anyhow::Result<()> {
+    let issue_event = if let Event::Issue(event) = event {
```

Comment:
This feature likely wants to be implemented with the more typical handler (like major change, autolabel) so that we can enable it only on particular repositories. I think the configuration would likely benefit from being a list - we likely want to support more than one label as the condition and more than one as the resulting addition on the PR.